### ISSUE TITLE
As a permit purchaser, I need to understand how much the wood I’d like to harvest will cost, so that I understand how much money I’ll need to make the purchase.

### ISSUE THREAD

**Issue by MelissaBraxton** [USDAForestService/USFS-timber-permitting#113]
**Background**
_What should we consider when addressing this issue?_

**Acceptance criteria** 
- [ ] 
- [ ] 

**Tasks**
- [ ] 
- [ ] 

**Definition of done**
- [ ]  
- [ ]

**Comment by MelissaBraxton** [comment]
We've mocked cost as calculated on the permit data entry form. [Here is the mock](https://github.com/USDAForestService/USFS-timber-permitting/blob/master/docs/design-deliverables/online-timber-permitting-ui/2019-12--mvp-usability-test/FS%20MVP1--6.0%20Order%20permit--permit%20details.jpg)

**Comment by carlsonem** [comment]
@Dmac26 has this been started?

**Comment by Dmac26** [comment]
@carlsonem - Hey Eric, I had a couple of false starts on this last week and a few other issues that came up. But, I'm firmly started today.

**Comment by jstrothman** [comment]
@Dmac26 It looks like this one's waiting on design review. Could you tag me here when it's on dev or staging? Thanks!

**Comment by Dmac26** [comment]
@jstrothman - I will let you know! There were a few code changes that needed to be made. Those are done now and I'm just working on getting the merge done. I'll keep you posted!

**Comment by jstrothman** [comment]
@Dmac26 I see this is ready for review, and the validation is working. 
Currently, we show the minimum, but not the maximum number of cords for the forest, so it's easy to put in too many and find out the constraints after input.

Can we change the language to:
**Number of cords ***
Cost: $5 per cord
You can get between 4 - 12 cords
[Input field]
**Total cost for firewood permit**

with an error message: 'Number of cords must be between x and y'

**Comment by jstrothman** [comment]
@Dmac26 Any update on this adjustment to showing the max cords and changing the error message?

**Comment by Dmac26** [comment]
@jstrothman - Sorry I missed your comment from earlier. Since this has already been merged up to dev, I'd like to make a new card for this and track there if you don't mind. This should be an easy fix though. I should have a Pull Request in by tomorrow morning.

**Comment by mtlaney** [comment]
Calculations appear to be correct on the open forest side. We will need to verify we are sending the right amount to pay.gov
